### PR TITLE
build: release: v1.28.2

### PR DESCRIPTION
**Pull request by jennijuju**
https://github.com/filecoin-project/lotus/issues/12379

This patch includes:

- [x] #12390  
- [x] #12395

### CODE REVIEW

**Comment by beck-8** [comment]
Hopefully v1.28.2 will completely fix the pledge issue together.

**Comment by MikeH1999** [comment]
Is it already available?

**Review by Stebalien — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,22 @@
 # Lotus changelog
 
+# 1.28.2 / 2024-08-15
+
+This is a Lotus patch release v1.28.2 for Node operators and Storage Providers.
+
+For node operators, this patch release fixes an issue where excessive bandwidth usage (issue #12381) was caused by a routing loop in pubsub, where small "manifest" messages were cycling repeatedly around the network due to an ineffective routing loop prevention mechanism.
+
+For storage providers this patch release fixes pledge issues, users have been encountering. This update addresses existing issues, including the too-small pledge in snap and the lack of DDO-awareness in PoRep Commit.
```
Comment (BigLep) [inline CHANGELOG.md:9]:
```suggestion
For storage providers, this patch release fixes pledge issues users have been encountering. This update addresses existing issues, including the too-small pledge in snap and the lack of DDO-awareness in PoRep Commit.
```

Also, do we have an issue we can link to?

**Comment by BigLep** [comment]
I think we should also make clear if we're going to backport additional fixes around pledging like what was proposed in https://github.com/filecoin-project/lotus/pull/12386

File: CHANGELOG.md (hunk 1 above)
Comment (jennijuju) [reply CHANGELOG.md:9]: seems like the reports didnt get from slack to github

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,22 @@
 # Lotus changelog
 
+# 1.28.2 / 2024-08-15
+
+This is a Lotus patch release v1.28.2 for Node operators and Storage Providers.
+
+For node operators, this patch release fixes an issue where excessive bandwidth usage (issue #12381) was caused by a routing loop in pubsub, where small "manifest" messages were cycling repeatedly around the network due to an ineffective routing loop prevention mechanism.
```
Comment (jennijuju) [inline CHANGELOG.md:7]:
```suggestion
For node operators, this patch release is HIGHLY RECOMMENDED as it fixes an issue where excessive bandwidth usage (issue #12381) was caused by a routing loop in pubsub, where small "manifest" messages were cycling repeatedly around the network due to an ineffective routing loop prevention mechanism. The new f3 release also has a couple performance improvements around CPU usage. (If you are curious about the progress of F3 testing, follow the updates [here](https://github.com/filecoin-project/lotus/discussions/12287#discussioncomment-10343447)).
```

**Comment by jennijuju** [comment]
> I think we should also make clear if we're going to backport additional fixes around pledging like what was proposed in #12386

IIUC we won't?

**Comment by BigLep** [comment]
> I think we should also make clear if we're going to backport additional fixes around pledging like what was proposed in #12386

I think I missed some of the details about backporting and what has been done is good.   https://github.com/filecoin-project/lotus/pull/12395, which was backported, was informed by https://github.com/filecoin-project/lotus/pull/12386

I don't see any issues at a high level.  Thanks for handling.

**Pull request merged**